### PR TITLE
Update Celer cBridge adapter

### PR DESCRIPTION
**Pull request by Dominator008**
Include more bridge contracts and tokens

### CODE REVIEW

File: projects/celerbridge/index.js [hunk 1]
```diff
@@ -84,16 +428,28 @@ function chainTvl(chain) {
...
-            })
+            let balanceV1;
+            try {
+                balanceV1 = await sdk.api.erc20.balanceOf({
+                    chain,
+                    block: block,
+                    target: token[chain],
+                    owner: bridgeContractV1
+                })
+            } catch (error) {}
```
Comment (0xngmi) [inline projects/celerbridge/index.js:439]: better to fail hard in case this fails

**Comment by 0xngmi** [comment]
why all the try/catches?

**Comment by Dominator008** [comment]
> why all the try/catches?

The test seems flaky when I ran it. Getting call exceptions pretty frequently.

File: projects/celerbridge/index.js (hunk 1 above)
Comment (Dominator008) [reply projects/celerbridge/index.js:439]: Was getting `CALL_EXCEPTION`s pretty frequently but not deterministically. Any suggestion?

**Comment by Dominator008** [comment]
> why all the try/catches?

Removed them. PTAL.

**Pull request merged**